mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -56,7 +56,10 @@ private[ml] trait CrossValidatorParams extends ValidatorParams {
 
 /**
  * :: Experimental ::
- * K-fold cross validation.
+ * CrossValidator begins by splitting the dataset into a set of folds which are 
+ * used as separate training and test datasets e.g., with k=3 folds, CrossValidator 
+ * will generate 3 (training, test) dataset pairs, each of which uses 2/3 of the data 
+ * for training and 1/3 for testing.
```

Comment:
It may be worth mentioning that each fold is non-overlapping, and that each fold will be used in test exactly once. See [wikipedia](https://en.wikipedia.org/wiki/Cross-validation_(statistics)#k-fold_cross-validation).